package.json

```diff
@@ -4,20 +4,39 @@
   "license": "Apache-2.0",
   "private": true,
   "engines": {
-    "node": ">=12"
+    "node": ">=20.0.0"
   },
   "workspaces": [
     "pkg/*"
   ],
   "dependencies": {
     "@rancher/components": "0.2.1-alpha.0",
-    "@rancher/shell": "^2.0.1",
-    "@types/lodash": "4.14.184",
```

Comment:
`lodash/isEmpty` package is still in use in source code, to the `@types/lodash` should be listed in the `dependencies`